src/ORM/Sluggable/SluggableSubscriber.php

```diff
@@ -93,4 +102,25 @@ private function isSluggable(ClassMetadata $classMetadata)
             $this->isRecursive
         );
     }
+
+    /**
+     * @param $entity
+     * @param EntityManager $em
+     */
+    private function generateUniqueSlugFor($entity, EntityManager $em)
+    {
+        $slug = $entity->getSlug();
+        $uniqueSlug = $slug;
+
+        $i = 0;
+        while ($em->getRepository(get_class($entity))->matching(
```

Comment:
@docteurklein What do you think about `matching()` method in this case? Is it an overhead? Maybe better to use a QueryBuilder here?